bin/ruby-build

```diff
@@ -206,6 +206,21 @@ is_freebsd() {
   [ "$(uname -s)" = "FreeBSD" ]
 }
 
+is_fedora() {
+  if [ -r /etc/os-release ]; then
+      # shellcheck disable=SC1091
+      source /etc/os-release
+      # This will treat Aurora, Bazzite, BlueFin, uCore, and other Fedora variants as `fedora`.
+      # shellcheck disable=SC2153
+      local os_id
+      od_id="${ID_LIKE:-$ID}"
+      local os_version
+      os_version=$VERSION_ID
+      [ "${od_id}" = "fedora" ] || return 1
+      [ $# -eq 0 ] || [ "${os_version}" -ge "$1" ]
```

Comment:
Nitpick: this could be simplified, something like
```suggestion
      if [[ "${ID_LIKE:-$ID}" != "fedora" || ( $# -gt 0 && "$VERSION_ID" -lt "$1") ]]; then
        return 1
      fi
```